Web Inspector: REGRESSION(?): Console: find results count has no left padding

dcrousso opened the pull request:
#### 347111d578a4171bc5ef8b5917ba41c4c22acc59
<pre>
Web Inspector: REGRESSION(?): Console: find results count has no left padding
<a href="https://bugs.webkit.org/show_bug.cgi?id=256531">https://bugs.webkit.org/show_bug.cgi?id=256531</a>

Reviewed by Patrick Angle.

* Source/WebInspectorUI/UserInterface/Views/FindBanner.css:
(.find-banner):
(.find-banner &gt; input[type=&quot;search&quot;]):
(.find-banner &gt; button):
(.find-banner &gt; button.segmented):
(.find-banner &gt; label):
(.find-banner &gt; label:not(:empty)): Added.
(.find-banner &gt; :first-child): Deleted.
(.find-banner &gt; :last-child): Deleted.
(.find-banner.console-find-banner): Deleted.
(body .find-banner.console-find-banner): Deleted.
(.find-banner.console-find-banner &gt; input[type=&quot;search&quot;]): Deleted.
(body[dir=ltr] .find-banner.console-find-banner &gt; input[type=&quot;search&quot;]): Deleted.
(body[dir=rtl] .find-banner.console-find-banner &gt; input[type=&quot;search&quot;]): Deleted.
(.find-banner.console-find-banner &gt; :is(input[type=&quot;search&quot;], button)): Deleted.
(.find-banner.console-find-banner &gt; input[type=&quot;search&quot;]::-webkit-textfield-decoration-container): Deleted.
(.find-banner.console-find-banner &gt; input[type=&quot;search&quot;]:focus, .find-banner.console-find-banner &gt; input[type=&quot;search&quot;]:focus ~ button, .find-banner.console-find-banner &gt; input[type=&quot;search&quot;]:not(:placeholder-shown), .find-banner.console-find-banner &gt; input[type=&quot;search&quot;]:not(:placeholder-shown) ~ button): Deleted.
* Source/WebInspectorUI/UserInterface/Views/LogContentView.css:
(.navigation-bar &gt; .item.find-banner.console): Added.
(:not(.console-drawer) &gt; .navigation-bar &gt; .item.find-banner.console): Added.
(.navigation-bar &gt; .item.find-banner.console &gt; input[type=&quot;search&quot;]): Added.
(.navigation-bar &gt; .item.find-banner.console &gt; :is(input[type=&quot;search&quot;], button)): Added.
(.navigation-bar &gt; .item.find-banner.console &gt; input[type=&quot;search&quot;]::-webkit-textfield-decoration-container): Added.
(.navigation-bar &gt; .item.find-banner.console &gt; input[type=&quot;search&quot;]:focus, .navigation-bar &gt; .item.find-banner.console &gt; input[type=&quot;search&quot;]:focus ~ button, .navigation-bar &gt; .item.find-banner.console &gt; input[type=&quot;search&quot;]:not(:placeholder-shown), .navigation-bar &gt; .item.find-banner.console &gt; input[type=&quot;search&quot;]:not(:placeholder-shown) ~ button ): Added.
(.console-find-banner): Deleted.
Avoid using `:first-child`/`:last-child` (and `!important`) as they have very little control over what kinds of situations they apply (e.g. we don&apos;t want to add padding to `label:empty`).
Don&apos;t add any outside `margin` as that&apos;s handled by the containing `WI.NavigationBar` (or adjacent `WI.NavigationItem`) unless we know for sure that it&apos;s the first/last `WI.NavigationItem` in the `WI.NavigationBar`.
Drive-by: Move styles to a more relevant location.

* Source/WebInspectorUI/UserInterface/Views/FindBanner.js:
(WI.FindBanner):
* Source/WebInspectorUI/UserInterface/Views/LogContentView.js:
(WI.LogContentView):

Drive-by: Rework optional parameters into `{...} = {}` to match existing pattern elsewhere.
Canonical link: <a href="https://commits.webkit.org/264167@main">https://commits.webkit.org/264167@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/f0610917b1ff947603a05e1007a5bd19ed9a5b5a

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/6915 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/7145 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/7328 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/8515 "Built successfully") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/32/builds/7161 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/8400 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/7084 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/10065 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/7037 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/15/builds/7704 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/6369 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [❌ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/8615 "Failed to checkout and rebase branch from PR 13645") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/39/builds/5084 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/6277 "Passed tests") | [❌ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/2/builds/8615 "Failed to checkout and rebase branch from PR 13645") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/6769 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/6365 "Passed tests") | [❌ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/2/builds/8615 "Failed to checkout and rebase branch from PR 13645") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/6847 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/17/builds/5614 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/6223 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/1643 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/10408 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/6601 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

patrickangle commented on Source/WebInspectorUI/UserInterface/Views/LogContentView.css:
```diff
@@ -61,6 +56,56 @@
+    border-bottom-left-radius: var(--border-radius-end);
+    border-top-right-radius: var(--border-radius-start);
+    border-bottom-right-radius: var(--border-radius-start);
+}
```
I think we can use the logical border radius properties here. e.g. `border-start-start-radius` (https://developer.mozilla.org/en-US/docs/Web/CSS/border-start-start-radius)

dcrousso replied:
actually turns out we don't really need these anymore cause of `--find-banner-search-box-border-radius-start`

dcrousso committed:
Web Inspector: REGRESSION(?): Console: find results count has no left padding
https://bugs.webkit.org/show_bug.cgi?id=256531

Reviewed by Patrick Angle.

* Source/WebInspectorUI/UserInterface/Views/FindBanner.css:
(.find-banner):
(.find-banner > input[type="search"]):
(.find-banner > button):
(.find-banner > button.segmented):
(.find-banner > label):
(.find-banner > label:not(:empty)): Added.
(.find-banner > :first-child): Deleted.
(.find-banner > :last-child): Deleted.
(.find-banner.console-find-banner): Deleted.
(body .find-banner.console-find-banner): Deleted.
(.find-banner.console-find-banner > input[type="search"]): Deleted.
(body[dir=ltr] .find-banner.console-find-banner > input[type="search"]): Deleted.
(body[dir=rtl] .find-banner.console-find-banner > input[type="search"]): Deleted.
(.find-banner.console-find-banner > :is(input[type="search"], button)): Deleted.
(.find-banner.console-find-banner > input[type="search"]::-webkit-textfield-decoration-container): Deleted.
(.find-banner.console-find-banner > input[type="search"]:focus, .find-banner.console-find-banner > input[type="search"]:focus ~ button, .find-banner.console-find-banner > input[type="search"]:not(:placeholder-shown), .find-banner.console-find-banner > input[type="search"]:not(:placeholder-shown) ~ button): Deleted.
* Source/WebInspectorUI/UserInterface/Views/LogContentView.css:
(.navigation-bar > .item.find-banner.console): Added.
(:not(.console-drawer) > .navigation-bar > .item.find-banner.console): Added.
(.navigation-bar > .item.find-banner.console > input[type="search"]): Added.
(.navigation-bar > .item.find-banner.console > :is(input[type="search"], button)): Added.
(.navigation-bar > .item.find-banner.console > input[type="search"]::-webkit-textfield-decoration-container): Added.
(.navigation-bar > .item.find-banner.console > input[type="search"]:focus, .navigation-bar > .item.find-banner.console > input[type="search"]:focus ~ button, .navigation-bar > .item.find-banner.console > input[type="search"]:not(:placeholder-shown), .navigation-bar > .item.find-banner.console > input[type="search"]:not(:placeholder-shown) ~ button ): Added.
(.console-find-banner): Deleted.
Avoid using `:first-child`/`:last-child` (and `!important`) as they have very little control over what kinds of situations they apply (e.g. we don't want to add padding to `label:empty`).
Don't add any outside `margin` as that's handled by the containing `WI.NavigationBar` (or adjacent `WI.NavigationItem`) unless we know for sure that it's the first/last `WI.NavigationItem` in the `WI.NavigationBar`.
Drive-by: Move styles to a more relevant location.

* Source/WebInspectorUI/UserInterface/Views/FindBanner.js:
(WI.FindBanner):
* Source/WebInspectorUI/UserInterface/Views/LogContentView.js:
(WI.LogContentView):

Drive-by: Rework optional parameters into `{...} = {}` to match existing pattern elsewhere.
Canonical link: https://commits.webkit.org/264167@main
